app/src/main/java/ar/com/wolox/android/example/ui/home/HomeActivity.kt

```diff
@@ -0,0 +1,27 @@
+package ar.com.wolox.android.example.ui.home
+
+import android.content.Context
+import android.content.Intent
+import ar.com.wolox.android.R
+import ar.com.wolox.wolmo.core.activity.WolmoActivity
+
+/**
+ * HomeActivity class
+ */
+class HomeActivity : WolmoActivity() {
+    override fun init() {
+        replaceFragment(R.id.vActivityBaseContent, HomeFragment.newInstance())
+    }
+
+    override fun layout(): Int {
+        return R.layout.activity_base
+    }
+
+    companion object {
+        @JvmStatic
```

Comment:
You don't need to use this annotation here. Everything inside the companion object, is consered static